packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -108,7 +112,8 @@ const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
 }) => {
   const showInsights = usePreference('showInsights', React);
   const editorContainerRef = useRef<HTMLDivElement>(null);
-  const editorRef = useRef<EditorRef>(null);
+  const editorInitialValueRef = useRef<string | false>(false);
+  const editorCurrentRef = useRef<EditorRef>(null);
```

Comment:
Any particular reason we renamed this? FWIW refs are supposed to hold the current value (hence the `ref.current` prop), so calling it `editorCurrentRef` makes this overly verbose. Feel free to ignore if this name makes more sense to you